docs/docs/legal/terms-of-service-dedicated.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Dedicated Instance Terms
+custom_edit_url: null
+--- 
+## General
+
+Last revised: July 20, 2021
+
+### Background
+
+Within the scope of the Framework Agreement, the Customer may choose to purchase a subscription that requires a dedicated instance of ZITADEL. These additional terms for dedicated instance ("**Dedicated Instance Terms**") apply in addition to the Framework Agreement.
+
+### Service
+
+CAOS operates and manages a **Dedicated Instance** of ZITADEL in a private infrastructure environment dedicated for Customer's use and provides support services for the customer according the Booking, these terms, agreed [**Service Level Description**](service-level-description), and [**Support Service Descriptions**](support-services).
```

Comment:
```suggestion
CAOS operates and manages a **Dedicated Instance** of ZITADEL in a private infrastructure environment dedicated for Customer's and provides support services for the Customer according the Booking, these terms, agreed [**Service Level Description**](service-level-description), and [**Support Service Descriptions**](support-services).
```

Reply:
Should be correct with the latest commit. Thanks.